CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 6.8.0
```

Comment:
Should it be a major version bump? It could be a breaking change.

Reply:
I'm fine with that - in reality I don't think any provider should have accepted this in the first place, but as we observed recently it seems to be provider specific, so this validation makes it a defined behaviour